presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDaoUtil.java

```diff
@@ -57,6 +57,22 @@ private static void createTables(SchemaDao dao)
         dao.createTableDeletedShards();
         dao.createTableBuckets();
         dao.createTableShardOrganizerJobs();
+
+        // for upgrading compatibility
+        alterTable(dao::alterTableTablesWithDeltaDelete);
+        alterTable(dao::alterTableTablesWithDeltaCount);
+        alterTable(dao::alterTableShardsWithIsDelta);
+        alterTable(dao::alterTableShardsWithDeltaUuid);
+    }
+
+    private static void alterTable(Runnable alterTableFunction)
+    {
+        try {
+            alterTableFunction.run();
+        }
+        catch (Exception e) {
+            log.info("Alter table failed: %s", e);
```

Comment:
`log.warn(e, "Alter table failed");`

Reply:
changed, but slightly different due to it producing too much log and cause test to fail